include/igl/kelvinlets.cpp

```diff
@@ -0,0 +1,208 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+//
+// Copyright (C) 2013 Alec Jacobson <alecjacobson@gmail.com>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+
+#include "kelvinlets.h"
+#include "PI.h"
+#include "parallel_for.h"
+
+constexpr double POISSON_RATIO = 0.5;
+constexpr double SHEAR_MODULUS = 1;
+constexpr double a = 1 / (4 * igl::PI * SHEAR_MODULUS);
```

Comment:
Can you define those locally, so as not to pollute the global namespace?

Reply:
Moved them